### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by spaulaus**
I tried to make sure that everything got attributed properly. I consulted the following repos to find information: 

* kmiernik/pixie_scan
* pixie16/PixieSuite
* pixie16/pixie_scan
* pixie16/PixieSuite2

### CODE REVIEW

File: Acquisition/MCA/share/mca_input.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# K. A. Miernik
```
Comment (ksmith0) [inline Acquisition/MCA/share/mca_input.txt:1]: Did this break the functionality?

File: Acquisition/Setup/source/input_whole_crate.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#@authors K. Miernik, K. Smith
```
Comment (ksmith0) [inline Acquisition/Setup/source/input_whole_crate.txt:1]: Does this break functionality?

File: Acquisition/MCA/share/mca_input.txt (hunk 1 above)
Comment (spaulaus) [reply Acquisition/MCA/share/mca_input.txt:1]: It shouldn't that will be recognized as a comment line.

File: Acquisition/Setup/source/input_whole_crate.txt (hunk 2 above)
Comment (spaulaus) [reply Acquisition/Setup/source/input_whole_crate.txt:1]: Again should be recognized as a comment line.

File: Acquisition/MCA/share/mca_input.txt (hunk 1 above)
Comment (ksmith0) [reply Acquisition/MCA/share/mca_input.txt:1]: Confirmed

**Review by ksmith0 — APPROVED** [review]
These changes are correct to my knowledge.

**Pull request merged**